app/forms/candidate_interface/becoming_a_teacher_form.rb

```diff
@@ -26,9 +26,18 @@ def self.build_from_params(params)
     end
 
     def save(application_form)
-      application_form.update(
-        becoming_a_teacher:,
-      )
+      ActiveRecord::Base.transaction do
+        application_form.update!(
+          becoming_a_teacher:,
+        )
+
+        application_form
+          .application_choices
+          .all? { |choice| choice.update!(personal_statement: becoming_a_teacher) }
+      end
+    rescue ActiveRecord::ActiveRecordError => e
+      errors.add(:becoming_a_teacher, I18n.t('.page_titles.internal_server_error'))
```

Comment:
Had a look through the [design system guidelines](https://design-system.service.gov.uk/components/error-message/) and they advise to only render these types of errors if the user has provided information we're unable to use. We need to follow this pattern if something goes wrong:

> Do not use error messages to tell a user that they are not eligible or do not have permission to do something. Or to tell them about a lack of capacity or other problem the user cannot fix - because the problem is with the service rather than with the information the user has provided.